.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,185 +2,193 @@ name: Bug Report
+              If you can't find a similar question on Stack Overflow, please create a new Stack Overflow thread using the tag `msal.js`.
+
+              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
+
+    - type: markdown
+      attributes:
+          value: |
+              ## Issue Details
+
+    - type: dropdown
+      attributes:
+          label: Core Library
+          options:
+              - "MSAL.js (@azure/msal-browser)"
+              - "MSAL.js v1 (@azure/msal or msal)"
+              - "MSAL Node (@azure/msal-node)"
+              - "Passport Azure AD (passport-azure-ad)"
+              - "ADAL Node (adal-node)"
+              - "ADAL.js (adal.js)"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Core Library Version
+          description: "Please enter the latest version this issue can be reproduced in."
+          placeholder: "3.1.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Wrapper Library
+          options:
+              - "MSAL Angular (@azure/msal-angular)"
+              - "MSAL React (@azure/msal-react)"
+              - "MSAL Node Extensions (@azure/msal-node-extensions)"
+              - "ADAL Angular (adal-angular)"
+              - "Not Applicable"
+      validations:
+          required: true
+
+    - type: input
+      attributes:
+          label: Wrapper Library Version
+          description: "Please enter the latest version this issue can be reproduced in. If you are not using a wrapper library please enter 'None'"
+          placeholder: "2.0.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Public or Confidential Client?
+          description: Are you using PublicClientApplication or ConfidentialClientApplication? (Issues regarding msal-browser, msal-react and msal-angular are always "Public")
+          multiple: true
+          options:
+              - "Public"
+              - "Confidential"
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Description
+          description: "Please briefly describe your issue."
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
+          label: Msal Logs
+          description: "Please provide verbose level log messages from msal. Bug reports without verbose MSAL logs will be closed without comment."
```

Comment:
```suggestion
          description: "Please provide verbose or trace level log messages from msal. Bug reports without logs will be closed without comment."
```

Reply:
Addind docs here for logging instructions would help.

Reply:
Done